website/source/docs/provisioners/powershell.html.md.erb

```diff
@@ -32,6 +32,13 @@ The example below is fully functional.
 ## Configuration Reference
 
 <%= partial "partials/provisioners/shell-config" %>
+-   `debug_mode` (int) - If set, sets PowerShell's [PSDebug mode](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/set-psdebug?view=powershell-7)
+    in order to make script debugging easier. For instance, setting the
+    value to 1 results in adding this to the execute command:
+
+    ``` powershell
+    Set-PSDebug -Trace 1
+    ```
```

Comment:
You can add this as a comment above the variable definition in `provisioner/powershell/provisioner.go`
```
// If set, sets PowerShell's [PSDebug mode](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/set-psdebug?view=powershell-7)
//  in order to make script debugging easier. For instance, setting the
//    value to 1 results in adding this to the execute command:
//
//    ``` powershell
//    Set-PSDebug -Trace 1
//    ```
DebugMode int `mapstructure:"debug_mode"`
```
Then you can run `make generate`  that the doc will be generated with what its commented above the variable. 
See about code generation in our [CONTRIBUTING docs](https://github.com/hashicorp/packer/blob/master/.github/CONTRIBUTING.md#code-generation)

Reply:
Thanks for that! I _believe_ I generated the docs correctly. I also see the original source doc I was editing has been deleted after PR 8940, so let me know if there's anything else I should do here. Thanks!

Reply:
It appears I've still done something wrong, as previewing the website deployment via https://5e975dc25ea217000742e310--packer-www.netlify.app/docs/provisioners/powershell/ doesn't show any entry mentioning `debug_mode`.

Should I manually edit `website/pages/docs/provisioners/powershell.mdx`?

 Any help is greatly appreciated. Thanks!

Reply:
Apparently this provisioner doesn't use our common doc generator. My mistake, sorry! 
I will add this later in another PR. For now, I've pushed the docs to your branch by just updating the powershell.mdx so I can merge this :)